app/scripts/views/settings/emails.js

```diff
@@ -73,7 +73,12 @@ define(function (require, exports, module) {
     },
 
     _onEmailCreateError (err) {
-      if (AuthErrors.is(err, 'EMAIL_ALREADY_EXISTS')) {
+      if (AuthErrors.is(err, 'EMAIL_EXISTS')
```

Comment:
I based this off of [sms](https://github.com/mozilla/fxa-content-server/blob/c18373771e2faa5d588f937e9ddcc457646151c2/app/scripts/views/sms_send.js#L159) error handling and it avoids showing unrelated errors to the user. Not sure how they could get into that state, so I don't mind removing either.